### PR TITLE
Add dynamic theme switching. Add theme colors. Add typography/font

### PR DESCRIPTION
**Pull request by defaultdino**

### CODE REVIEW

File: app/src/main/java/android/app/lotus/view/auth/Login.kt [hunk 1]
```diff
@@ -36,21 +35,11 @@ import androidx.compose.ui.unit.sp
...
 
 @Composable
-fun Login(navController: NavHostController, authViewModel: AuthViewModel) {
+fun Login(navController: NavHostController, mainViewModel: MainViewModel) {
     Box (
         modifier = Modifier.fillMaxSize(),
         contentAlignment = Alignment.BottomCenter
     ) {
-
-        Image(
```
Comment (PsychicPlatypus) [inline app/src/main/java/android/app/lotus/view/auth/Login.kt:45]: Why remove the image?

**Review by Quinhook — APPROVED** [review]

File: app/src/main/java/android/app/lotus/view/general/TopBar.kt [hunk 2]
```diff
@@ -1,32 +1,68 @@
...
+            ) {
+
+                ThemeSwitch(isDarkTheme, updateTheme)
+            }
         },
     )
 }
+
+@Composable
+fun ThemeSwitch(isDarkTheme: Boolean, updateTheme: (Boolean) -> Unit) {
```
Comment (PsychicPlatypus) [inline app/src/main/java/android/app/lotus/view/general/TopBar.kt:49]: Huge thumbs up!

**Review by PsychicPlatypus — APPROVED** [review]
Some comments on image removal

File: app/src/main/java/android/app/lotus/view/auth/Login.kt (hunk 1 above)
Comment (defaultdino) [reply app/src/main/java/android/app/lotus/view/auth/Login.kt:45]: When switching to dark mode the image wasn't very visible (due to there being dark parts in the image). We can find a better image and use it instead in the future ;)

**Pull request merged**